src/lv_interop.cc

```diff
@@ -51,6 +52,9 @@ namespace grpc_labview
     //---------------------------------------------------------------------
     void InitCallbacks()
     {
+        // Instantiating the feature toggles singleton that will read the feature configuration file
+        FeatureConfig::getInstance().readConfigFromFile("feature_config.ini");
```

Comment:
This seems to be a correct place to call readConfig . InitCallbacks() is the first method to get called while we do CreateClient, CreateServer, CreateSerializationSression etc.

Reply:
Removed it in all other locations.